src/include/txn_inline.h

```diff
@@ -917,8 +918,14 @@ __wt_upd_alloc(WT_SESSION_IMPL *session, const WT_ITEM *value, u_int modify_type
      *    WT_UPDATE.start_ts = WT_TS_NONE;
      *    WT_UPDATE.prepare_state = WT_PREPARE_INIT;
      *    WT_UPDATE.flags = 0;
+     *
+     * If there is no value ensure that the memory allocated is never less than the size
+     * reported by sizeof(). Otherwise bit-exact tools like MSan may infer the structure is not
+     * completely initialized.
```

Comment:
Could you please move this comment up with the macro above, where there is a block of things handling the oddities around WT_UPDATE data structures.

Reply:
Restored previous order.